core/types/src/lib.rs

```diff
@@ -25,7 +25,7 @@ pub use crate::{
     config_id::ConfigId,
     error::{Error, FfiError},
     key_request::{KeyName, KeyPolicy, KeyRequest, KeyRequestBuilder},
-    measurement::{Measurement, MrEnclave, MrSigner},
+    measurement::{MrEnclave, MrSigner},
```

Comment:
Note the module is still named `measurement`, up for suggestions on renaming if desired